app/src/main/java/com/asfoundation/wallet/ui/iab/LocalPaymentInteractor.kt

```diff
@@ -76,6 +78,16 @@ class LocalPaymentInteractor(private val deepLinkRepository: InAppDeepLinkReposi
     inAppPurchaseInteractor.saveAsyncLocalPayment(paymentMethod)
   }
 
+  fun showSupport(gamificationLevel: Int): Completable {
```

Comment:
- [ ] Avoid using explicit default locales

- [ ] can be converted to a expression body

Reply:
Don't agree with the second one